docs/api/backend.rst

```diff
@@ -6,7 +6,4 @@ Backend
    :members:
 
 .. automodule:: geomstats._backend.pytorch
-   :members:
```

Comment:
This should have autograd too :( Feel free to either add it, or leave an issue on parsing the library for missing mentions of autograd.